packages/react-scripts/template/README.md

```diff
@@ -1627,7 +1627,32 @@ script:
 
 #### CircleCI
 
-Follow [this article](https://medium.com/@knowbody/circleci-and-zeits-now-sh-c9b7eebcd3c1) to set up CircleCI with a Create React App project.
+1. Add a folder `.circleci` to project root.
+2. Add a `config.yml` inside it.
+
+```
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:6
```

Comment:
Should we update this to `circleci/node:8` or a similar updated tag?

Reply:
yes, agree. I updated it